static/js/controllers/edit-user.js

```diff
@@ -125,7 +125,7 @@
       };
 
       var getRoles = function(type, includeAdmin) {
-        if (includeAdmin && type === 'admin') {
+        if (includeAdmin && type === '_admin') {
```

Comment:
So this and every other instance of me replacing `admin` with `_admin` is because that's the role that the admin user has for both me and LG. I'm not if that means we're broken or the code was. Your thoughts, as always, are appreciated.